header/dex.go

```diff
@@ -0,0 +1,61 @@
+package header
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/moov-io/iso8583/encoding"
+)
+
+var _ Header = (*VisaDexHeader)(nil)
+
+// Visa DEX header is 2 bytes of BCD encoded length
```

Comment:
Could we just make this a 2ByteHeader. I know Visa DEX is only two bytes but Discover and others are the same and that makes it seem strange to use the Visa name for other networks. Visa also probably crawls the internet looking for their name.

Reply:
Done!

I think the next step can be to add CustomHeader:
```go
header := NewCustomHeader(CustomHeaderOptions{
  bytes: 2,
  enc: encoding.BCD,
})
```

then all other headers can be implemented via Custom.